include/grpcpp/security/credentials_impl.h

```diff
@@ -129,6 +129,14 @@ class CallCredentials : private grpc::GrpcLibraryCodegen {
   virtual SecureCallCredentials* AsSecureCredentials() = 0;
 };
 
+struct TlsVersion {
```

Comment:
Consider calling this struct `TlsVersions` or `TlsVersionRange`, since it does more than store a single TLS version.